packaging/debian/control

```diff
@@ -3,7 +3,6 @@ Section: admin
 Priority: optional
 Maintainer: Cockpit <cockpit-devel@lists.fedorahosted.org>
 Build-Depends: debhelper-compat (= 12),
-               appstream-util,
```

Comment:
This commit `debian: Drop appstream-util check ` could be merged into the previous one as it directly fixes it.

Reply:
That was by intent -- it's easier to revert once we get appstream-util into the Debian image's pbuilder.